internal/controller/stas/containerimagescan_status.go

```diff
@@ -28,8 +36,80 @@ func newContainerImageStatusPatch(cis *stasv1alpha1.ContainerImageScan) *stasv1a
 		}
 	}
 
-	return stasv1alpha1ac.ContainerImageScan(cis.Name, cis.Namespace).
-		WithStatus(status)
+	return &containerImageScanStatusPatch{
+		cis: cis,
+		patch: stasv1alpha1ac.ContainerImageScan(cis.Name, cis.Namespace).
+			WithStatus(status),
+	}
+}
+
+type containerImageScanStatusPatch struct {
+	cis             *stasv1alpha1.ContainerImageScan
+	patch           *stasv1alpha1ac.ContainerImageScanApplyConfiguration
+	vulnerabilities []stasv1alpha1.Vulnerability
+	minSeverity     *stasv1alpha1.Severity
+}
+
+func (b *containerImageScanStatusPatch) withCondition(c *metav1ac.ConditionApplyConfiguration) *containerImageScanStatusPatch {
```

Comment:
Why `b` as receiver name? :D

Reply:
From the previous name, ....Builder. Fixed now.